### ISSUE TITLE
Add a default `{tlf}` theme for `{ospsuite}` plots

### ISSUE THREAD

**Issue by IndrajeetPatil** [Open-Systems-Pharmacology/OSPSuite-R#829]
# Task

Add a default `{tlf}` theme for `{ospsuite}` plots

# Background

_tlf_ package provides [`Theme`](https://www.open-systems-pharmacology.org/TLF-Library/reference/Theme.html) objects, which provide a single entry point to set up the _look_ of the plot by choosing:

- aesthetic mappings (what variables are mapped to `colour`, `fill`, `linetype`, etc.)
- background (watermark, legend position, grid, etc.)
- properties (colour, font, size, etc.) for the plot text (title, subtitle, legend, axes labels, etc.)

These objects are **not** involved in setting up scale transformations; that's the job of the [`Scaling` ](https://www.open-systems-pharmacology.org/TLF-Library/reference/Scaling.html)object.

As decided in our relevant meetings, we **don't** want, as much as possible, for users to be exposed to the underlying _tlf_ `R6` objects. Therefore, we will choose a theme (and thereby a _look_) in a way that satisfies most users' expectations and aesthetic preferences out of the box. 

In other words, we will try our best to include a default theme for all plotting functions that circumvents users ever having to learn how to create an instance of the `Theme` class. And, of course, those who want to change the theme will be provided with an option to provide their own home-grown `Theme` instance.

For more, see:
https://www.open-systems-pharmacology.org/TLF-Library/articles/theme-maker.html
